pkgs/top-level/python-packages.nix

```
@@ -1321,6 +1321,8 @@ in {
 
   cozy = callPackage ../development/python-modules/cozy { };
 
+  CppHeaderParser = callPackage ../development/python-modules/CppHeaderParser { };
```

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

```suggestion
  cppheaderparser = callPackage ../development/python-modules/CppHeaderParser { };
```